lib/src/showcase.dart

```diff
@@ -359,14 +355,12 @@ class _TargetWidget extends StatelessWidget {
             height: size!.height + 16,
             width: size!.width + 16,
             decoration: ShapeDecoration(
-              shape: radius != null
```

Comment:
This removes the one of the functionality. please undo this changes.

Reply:
It was not used anywhere and has this warning from lint
`A value for optional parameter 'radius' isn't ever given.`

I did undo this anyway.

Reply:
Hi @P-B1101 , we have fixed one of the github workflow, which was failing the GitHub Actions, so could you please take `rebase / update` from master branch to this branch ?

Reply:
Hi @vatsaltanna, I'm actually not an expert on git. Did I do it right?